authorization/invitation/service/invitation_service_blackbox_test.go

```diff
@@ -35,29 +36,35 @@ func (s *invitationServiceBlackBoxTest) SetupTest() {
 }
 
 func (s *invitationServiceBlackBoxTest) TestIssueInvitationByIdentityID() {
-	// Create a test user - this will be the organization owner
-	identity, err := test.CreateTestIdentityAndUserWithDefaultProviderType(s.DB, "invitationServiceBlackBoxTest-TestIssuingUser")
-	require.Nil(s.T(), err, "Could not create identity")
+	g := s.NewTestGraph()
```

Comment:
Just a note. You could use already available s.Graph instead of creating it explicitly manually in your tests via s.NewTestGraph(). We create a new graph for every test function in DBSuite - https://github.com/fabric8-services/fabric8-auth/blob/master/gormtestsupport/db_test_suite.go#L70
Example of usage - https://github.com/fabric8-services/fabric8-auth/blob/master/authorization/role/service/role_management_service_blackbox_test.go#L45